src/component/create.modal.tsx

```diff
@@ -0,0 +1,123 @@
+"use client";
+import { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import { toast } from "react-toastify";
+import { mutate } from "swr";
+
+interface IProps {
+  showModalCreate: boolean;
+  setShowModalCreate: (v: boolean) => void;
+}
+
+function CreateModal(props: IProps) {
+  const { showModalCreate, setShowModalCreate } = props;
+  const [title, setTitle] = useState<string>("");
+  const [author, setAuthor] = useState<string>("");
+  const [content, setContent] = useState<string>("");
+
+  const handleSubmit = () => {
+    if (!content && !content && !author) {
+      toast.error("Not empty data !");
+      return;
+    }
+    if (!title) {
+      toast.error("Not empty title !");
+      return;
+    }
+    if (!author) {
+      toast.error("Not empty author !");
+      return;
+    }
+
+    if (!content) {
+      toast.error("Not empty content !");
+      return;
+    }
+
+    fetch("https://652c0736d0d1df5273ef0e4e.mockapi.io/api/v1/blogs", {
+      method: "POST",
+      headers: {
+        Accept: "application/json, text/plain, */*",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ title, author, content }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res) {
+          toast.success("Create new blogs succeed");
+          handleCloseModal();
+          mutate("https://652c0736d0d1df5273ef0e4e.mockapi.io/api/v1/blogs"); // gọi lại data để show ra mà không cân reload
+        }
+      });
+    // toast.success("Create succeed!...");
+    // console.log("check data form", title, author, content);
```

Comment:
remove these line

Reply:
Thank you for your feedback and I have removed those line